### DISCUSSION TITLE
Could not load file or assembly Magick.Native-Q8-x64.dll or one of its dependencies. The module was expected to contain an assembly manifest.

### DISCUSSION THREAD

**Discussion by romias** [dlemstra/Magick.NET#1382] [Help]
I'm trying to include Magick.NET into a web application running .NET Fwk 4.7.2 on a Windows 10 x64.
The error appears when running the application not even the code that uses Magick.net

Error: 
`Could not load file or assembly Magick.Native-Q8-x64.dll or one of its dependencies. The module was expected to contain an assembly manifest.`
In the stack trace it shows "BadImageFormatException".

The solution is running in AnyCPU configuration, I tried installing the AnyCPU package with that same error, only that it fails in the Arm version. 

Checked if the solution has the Prefer32bits set to true, and it is not.
The application pool on the IIS doesn't have Prefer32bits set.

I tried creating a Console app using the same framework version, and it works.

Any suggestion on what else to test?

**Comment by dlemstra** [comment]
You are getting `BadImageFormatException` when you try to run a 64-bit application in a 32-bit environment. When you use the `x64` version of Magick.NET you should make sure your application is running in 64-bit mode.
